tests/AlignedSegment_test.py

```diff
@@ -78,6 +78,12 @@ def testSettingTagInEmptyRead(self):
     def testCompare(self):
         '''check comparison functions.'''
         a = self.build_read()
+        b = None
+
+        self.assertFalse(a is b)
+        self.assertFalse(a == b)
+        self.assertFalse(a.compare(b))
```

Comment:
These two operations segfaulted immediately, and terminated the running process hard.  No other unit tests or code runs after this error is encountered.